src/document.jl

```diff
@@ -111,8 +111,8 @@ function save_file(xdoc::XMLDocument, filename::AbstractString; encoding::Abstra
 end
 
 function Base.string(xdoc::XMLDocument; encoding::AbstractString="utf-8")
-    buf_out = Vector{Xstr}(uninitialized, 1)
-    len_out = Vector{Cint}(uninitialized, 1)
+    buf_out = Vector{Xstr}(undef, 1)
```

Comment:
this needs a higher Compat min version

Reply:
thanks, 0.60.0 seem to be the first one. updated REQUIRE.

Reply:
0.59 https://github.com/JuliaLang/Compat.jl/commit/1862df279cf0f3e6afbf8712989a7c567cbc5171